graph.h

```diff
@@ -0,0 +1,27 @@
+#include <fstream>
+#include <vector>
+
+#include "town.h"
+#include "road.h"
+
+
+using std::ifstream;
+using std::vector;
+
+
+class graph
+{
+public:
+        graph();
+
+        void addTown(string name, bool capital);
+        void addRoad(string townOneString, string townTwoString, int distance, bool bridge);
+        void printTest();
+
+        void connectRoadsTowns();
```

Comment:
The comment in graph.cc about what this does really belongs here.  Elaborating a little more would be helpful.